packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -762,6 +780,22 @@ class FlameChartNode<T> {
     }
   }
 
+  Color _backgroundColor(bool selected) {
+    if (selected) return _selectedNodeColor;
+    if (useAlternateBackground != null && useAlternateBackground(data)) {
+      return alternateBackgroundColor;
+    }
+    return backgroundColor;
+  }
+
+  Color _textColor(bool selected) {
+    if (selected ||
+        (useAlternateBackground != null && useAlternateBackground(data))) {
+      return Colors.black;
```

Comment:
document why this should be black or add a semantically named constant for what this color is.

Reply:
done